File: tests/integration/construct_component_slow/python/component.py
# Copyright 2016-2021, Pulumi Corporation.  All rights reserved.

from typing import Any, Optional

import khulnasoft

class Component(khulnasoft.ComponentResource):
    def __init__(self, name: str, opts: Optional[khulnasoft.ResourceOptions] = None):
        props = dict()
        super().__init__("testcomponent:index:Component", name, props, opts, True)

File: tests/integration/construct_component_slow/nodejs/component.ts
// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.

import * as khulnasoft from "@khulnasoft/khulnasoft";

export class Component extends khulnasoft.ComponentResource {
    constructor(name: string, opts?: khulnasoft.ComponentResourceOptions) {
        super("testcomponent:index:Component", name, {}, opts, true);
    }
}
